source/TestContext.h

```diff
@@ -28,13 +28,24 @@ friend class Test;
 
 
 private:
-	// Pointer to the test we are running.
-	std::vector<const Test *> testToRun;
+	// Struct to describe a running test and running test-step within the test.
+	struct ActiveTestStep{
+		const Test *test;
+		unsigned int step;
+
+		// Support operators for usage in containers like map and set.
+		bool operator==(const ActiveTestStep &rhs) const;
+		bool operator<(const ActiveTestStep &rhs) const;
+		bool operator>(const ActiveTestStep &rhs) const;
```

Comment:
Maps and sets only need the `<` operator.

Reply:
That is true. Do you feel I should remove the `>` operator?

Reply:
Yes. Command also only has a `<` operator for maps/sets. IMO you could even remove the `==` operator since it isn't used as well.

Reply:
A set doesn't contain duplicates of the same element; so I expect `==` to be required for sets (unless it uses the `<` operator twice in a different order to determine if two elements are equal).

Reply:
I'll remove the `>` operator.

Reply:
Sets do use the < operator for equality, they don't need ==.